cluster-autoscaler/cloudprovider/azure/azure_scale_set.go

```diff
@@ -511,50 +534,113 @@ func (scaleSet *ScaleSet) Nodes() ([]cloudprovider.Instance, error) {
 	splay := rand.New(rand.NewSource(time.Now().UnixNano())).Intn(scaleSet.instancesRefreshJitter + 1)
 	lastRefresh := time.Now().Add(-time.Second * time.Duration(splay))
 
+	orchestrationMode, err := scaleSet.getOrchestrationMode()
+
+	if err != nil {
+		klog.Errorf("failed to get information for VMSS: %s, error: %v", scaleSet.Name, err)
+		return nil, err
+	}
+
+	klog.V(4).Infof("VMSS: orchestration Mode %s", orchestrationMode)
+
+	if orchestrationMode == compute.Uniform {
+		err := buildScaleSetCache(scaleSet, lastRefresh)
+
+		if err != nil {
```

Comment:
Remove empty line before the error check to follow the existing style

Reply:
Fixed